app.py

```diff
@@ -171,9 +169,7 @@ def _schedule_default_packages_solver_jobs(packages: List[str], index_urls: List
 
                 for version in versions:
                     _LOGGER.info("Scheduling package_name %r in package_version %r", package_name, version)
-                    number_workflows = _do_schedule_solver_jobs(
-                        openshift, index_urls, package_name, version, _SOLVER_OUTPUT
-                    )
+                    number_workflows = _do_schedule_solver_jobs(openshift, index_urls, package_name, version)
```

Comment:
In `_do_schedule_solver_jobs` method you have to remove also `output` which is not required anymore in `schedule_all_solvers`.